source/cl/test/UnitCL/source/ktst_precision.cpp

```diff
@@ -107,11 +107,9 @@ T Remquo7BitRef(T x, T y, cl_int &quo_out) {
 }
 }  // namespace
 
-#ifdef __arm__
-// TODO: CA-2730
-TEST_P(Execution, DISABLED_Precision_01_Pow_Func) {
-#elif defined(_WIN32)
-// TODO: CA-3492
+#if defined(__arm__) || defined(_WIN32) || defined(__APPLE__)
```

Comment:
totally understand why you've done this, but it feels a shame to not be able to run this test in some way on these architectures.

Reply:
Until we can fix the test to work properly, we can force it to run, if wanted, using the `--gtest_also_run_disabled_tests` option. It will require a rename of the kernel file (`precision.01_pow_func.cl` -> `disabled_precision.01_pow_func.cl`).